pkg/apis/serving/v1beta1/revision_defaults.go

```diff
@@ -44,6 +43,12 @@ func (rs *RevisionSpec) SetDefaults(ctx context.Context) {
 		rs.TimeoutSeconds = &ts
 	}
 
+	// Default ContainerConcurrency based on our configmap
+	if rs.ContainerConcurrency == nil {
+		cc := cfg.Defaults.ContainerConcurrency
+		rs.ContainerConcurrency = cc
```

Comment:
See my comment above, but this should end up being `ptr.Int64(cfg.Defaults.ContainerConcurrency)`